Client/src/main.jsx

```diff
@@ -2,9 +2,12 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import './index.css'
+import { BrowserRouter as Router, HashRouter } from 'react-router-dom'
```

Comment:
HashRouter unused at this time

Reply:
The reason I put HashRouter is that route location is not detected when we use useLocation hook, I thought maybe we could switch it later for a solution if we encounter an issue.

Reply:
Got it, thanks for the explanation :+1:

Reply:
Thanks for the review 👍